src/BuildScriptGeneratorCli/Options/NodeScriptGeneratorOptionsSetup.cs

```diff
@@ -25,6 +25,7 @@ public void Configure(NodeScriptGeneratorOptions options)
             options.CustomRunBuildCommand = GetStringValue(SettingsKeys.CustomRunBuildCommand);
             options.PruneDevDependencies = GetBooleanValue(SettingsKeys.PruneDevDependencies);
             options.NpmRegistryUrl = GetStringValue(SettingsKeys.NpmRegistryUrl);
+            options.EnableNodeMonorepoBuild = GetBooleanValue(SettingsKeys.EnableNodeMonorepoBuild);
```

Comment:
Since this is `false` by default, shouldn't you update your `lerna` and `lage` tests to make this `true` for those to pass?

Reply:
Also could you please run a Nightly build?